CarShowRoom/CarShowRoom/settings.py

```diff
@@ -27,6 +27,8 @@
 
 SHOW_SWAGGER = True if os.getenv("SHOW_SWAGGER", "False") == "True" else False
 
+APPEND_SLASH = False
```

Comment:
Where do you use it?

Reply:
It is used for routers. For ex: without that setting django raises an exception for using url like 0.0.0.0:8000/api/v1/cars/car/10
(without slash in the end), with APPEND_SLASH = False it will raise usual 404